.circleci/config.pre.yml

```diff
@@ -257,7 +257,7 @@ jobs:
         mkdir -p /tmp/workspace/artifacts /tmp/test_results /tmp/workspace/record /tmp/artifacts
     - run:
         command: |
-          cd .circleci && ./example.sh <<parameters.connector>> "${CIRCLE_NODE_TOTAL}" "${CIRCLE_NODE_INDEX}"
+          cd .circleci && ./run-examples.py <<parameters.connector>>
```

Comment:
The reason I originally had it the other way was so I could test it without setting the environment variables